src/common/config_opts.h

```diff
@@ -1217,8 +1217,16 @@ OPTION(bluestore_debug_random_read_err, OPT_DOUBLE, 0)
 OPTION(kstore_max_ops, OPT_U64, 512)
 OPTION(kstore_max_bytes, OPT_U64, 64*1024*1024)
 OPTION(kstore_backend, OPT_STR, "rocksdb")
+
 OPTION(kstore_rocksdb_options, OPT_STR, "compression=kNoCompression")
-OPTION(kstore_rocksdb_bloom_bits_per_key, OPT_INT, 0)
+OPTION(kstore_rocksdb_bloom_bits_per_key, OPT_INT, 20)
+OPTION(kstore_rocksdb_cache_index_and_filter_blocks, OPT_BOOL, false)
+OPTION(kstore_rocksdb_cache_index_and_filter_blocks_with_high_priority, OPT_BOOL, false)
+OPTION(kstore_rocksdb_index_type, OPT_STR, "binary_search")
+OPTION(kstore_rocksdb_partition_filters, OPT_BOOL, false)
+OPTION(kstore_rocksdb_metadata_block_size, OPT_U64, 0)
+OPTION(kstore_rocksdb_pin_l0_filter_and_index_blocks_in_cache, OPT_BOOL, false)
```

Comment:
So we are already using kstore_bloom_bits_per_key to set the bloom bits generally for rocksdb in kv/RocksDBStore.cc.  That's why I used the kstore prefix to match what we were already doing.  Should we change that as well?